app/javascript/dashboard/components/widgets/conversation/Message.vue

```diff
@@ -143,6 +143,9 @@ export default {
       return `https://twitter.com/${screenName}`;
     },
     alignBubble() {
+      if (this.data.message_type === 2) {
```

Comment:
Instead of using `2`, we have constants at `import { MESSAGE_TYPE } from 'shared/constants/messages';`. Please use them.